Test Utilities/Doubles/Dependencies.swift

```diff
@@ -0,0 +1,25 @@
+import XCTest
+@testable import PusherChatkit
+
+let DummyInstanceLocator = "dummy:instance:locator"
```

Comment:
There's already a test instance locator defined in an extension to `XCTest`: `Networking.testInstanceLocator`. It would be good to merge both into one.

Reply:
Agreed, we should only have one, I will attempt to merge.  I think we will need `valid` and `invalid` versions of the instance locators actually.

Reply:
FYI this is now resolved.  Because I've switched all my stuff to depend on the `InstanceLocator` entity I don't ever need to use a string.  The string is now removed.